Dockerfile.v2

```diff
@@ -0,0 +1,50 @@
+FROM radixdlt/babylon-node-build-layers:v1.3.0-java AS java-container
+
+WORKDIR /radixdlt
+
+# Copy the relevant files at the repo root
+COPY \
+  build.gradle \
+  gradlew \
+  gradlew.bat \
+  settings.gradle \
+  sonar-project.properties \
+  gradle.properties \
+  licence-header.txt \
+  /radixdlt/
+COPY ./gradle /radixdlt/gradle
+COPY ./common /radixdlt/common
+COPY ./core /radixdlt/core
+COPY ./core-rust-bridge /radixdlt/core-rust-bridge
+COPY ./cli-tools /radixdlt/cli-tools
+COPY ./shell /radixdlt/shell
+COPY ./keygen /radixdlt/keygen
+# Need .git for tag versions - but this can probably be removed soon
+COPY ./.git/* /radixdlt/.git/
+
+RUN SKIP_NATIVE_RUST_BUILD=TRUE ./gradlew clean build -x test -Pci=true -PrustBinaryBuildType=release
+
+RUN cd core/build/distributions && \
+    unzip -j *.zip && \
+    mkdir -p /artifacts && \
+    cp *.jar /artifacts && \
+    cp core /artifacts
+
+FROM radixdlt/babylon-node-build-layers:v1.3.0-rust AS rust-container
+
+WORKDIR /app
+
+COPY core-rust/ /app
+
+RUN mkdir -p /artifacts && \
+    /root/.cargo/bin/cargo build --profile=release && \
```

Comment:
Given we set `ENV PATH="$PATH:/root/.cargo/bin"` in the rust dockerfile, perhaps we can just do `cargo build` here, to better align across the two builds?